src/main/resources/entity-types/configuration/simple_config_data.json5

```diff
@@ -156,6 +170,20 @@
       queryable: true,
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->'metadata'->>'updatedByUserId'",
+      joinsTo: [
+        {
+          targetId: 'bb058933-cd06-4539-bd3a-6f248ff98ee2', // simple_user_details
```

Comment:
I would love for us to be able to replace entity type IDs within definitions like this with just the names of the entity type...but that's just a dream for now...